crawl-ref/source/job-data.h

```diff
@@ -77,6 +77,17 @@ static const map<job_type, job_def> job_data =
       { SK_THROWING, 2 }, { SK_WEAPON, 2 }, },
 } },
 
+{ JOB_BATTLEMAGE, {
+    "Ba", "Battlemage",
+    4, 4, 4,
+    { SP_TENGU, SP_KOBOLD, SP_BASE_DRACONIAN, SP_NAGA, SP_TROLL,
+      SP_MERFOLK, SP_DEMONSPAWN, },
+    { "leather armour plus:1", "book of War Wizardry" },
+    WCHOICE_NONE,
+    { { SK_FIGHTING, 1}, { SK_UNARMED_COMBAT, 3 }, { SK_DODGING, 2 },
+      { SK_SPELLCASTING, 2 }, { SK_TRANSMUTATIONS, 1 }, { SK_HEXES, 1 }, },
```

Comment:
one issue with your idea is that book backgrounds are already in some ways "better" than non book backgrounds.

Obviously an xxFi dies less frequently to random adder/gnoll ambushes, especially in the hands of unskilled players, but warrior backgrounds have sometimes replaced all their starting gear by D4, whereas books can be useful through most of lair before you start craving another book to eat.

it's cool and has that "remove/consolidate" meme going for it, but it also will require nerfing those roles, or buffing non-books. Or just removing Warrior classes and giving everyon books, since hybrids are generally best in crawl, lol?